packages/ui/src/1_atoms/Heading/Heading.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react';
+
+import React from 'react';
+
+import { Heading } from './Heading';
+import { HeadingType } from './Heading.types';
+
+describe('Heading', () => {
```

Comment:
Add a test that checks whether a heading with a simple string of content renders as expected - e.g something like:

`expect(h1).toHaveTextContent('content check')`